examples/example_launch_training.py

```diff
@@ -83,6 +94,16 @@ def create_parser():
         "output_prediction_horizon": args.output_prediction_horizon,
     }
 
+    input_robot_data_spec: RobotDataSpec = {
+        "223185e0-5250-487a-a7e7-c6a4b1d8c4e2": {
```

Comment:
this can be hard coded as each usr will have a different id. We need to get the robot id, perhaps from name?